docs/source/migration/3.0.mdx

```diff
@@ -725,6 +725,11 @@ Apollo Android 3.x instead uses `WebsocketNetworkTransport` to expose a simplifi
 - `protocolFactory` allows using `graphql-ws` or AppSync (or your custom one) as a lowlevel protocol instead of the default.
 - `reopenWhen` allows to automatically re-subscribe in case of a network error or another error.
 
-If you were previously using `apolloClient.subscriptionManager.reconnect()` to force a Websocket reconnect (for an example to send new authentication parameters), you should now create a new `ApolloClient`.
-
-If you feel some API is missing from `SubscriptionManager`, please [reach out](https://github.com/apollographql/apollo-kotlin/issues/new?assignees=&labels=%3Asparkles%3A+Type%3A+Feature&template=feature_request.md&title=) so we can discuss the best way to add them.
+If you were previously using `apolloClient.subscriptionManager.reconnect()` to force a Websocket reconnect (for example
+to send new authentication parameters), you can now
+call `apolloClient.subscriptionNetworkTransport.closeConnection(<your exception>)`: this will behave as if a network
+error occurred, and you can react to this by reconnecting with `reopenWhen`.
+
+If you feel some API is missing from `SubscriptionManager`,
+please [reach out](https://github.com/apollographql/apollo-kotlin/issues/new?assignees=&labels=%3Asparkles%3A+Type%3A+Feature&template=feature_request.md&title=)
+so we can discuss the best way to add them.
```

Comment:
This has to be in a separate PR so we can merge them separately

Reply:
Woops you're right of course!